### ISSUE TITLE
bunx solidui-cli fails

### ISSUE THREAD

**Issue by lly0414** [oven-sh/bun#12618]
### Stack Trace (bun.report)

Bun v1.1.20 ([`ae19489`](<https://github.com/oven-sh/bun/tree/ae19489250ab74ee3ea7d5b50fca5d7d2b4e6d66>)) on windows x86_64 [AutoCommand]

Segmentation fault at address 0x00000005

- *2 unknown/js code*

**Comment by lly0414** [comment]
============================================================
Bun v1.1.20 (ae194892) Windows x64
CPU: sse42 avx avx2
Args: "bun" "C:\Users\10275\AppData\Local\Temp\bunx-2192966315-solidui-cli@latest\node_modules\solidui-cli\dist\index.js" "init"
Features: jsc Bun.stdin(2) Bun.stdout bunfig tsconfig_paths tsconfig(21) 
Builtins: "bun:main" "node:assert" "node:buffer" "node:child_process" "node:events" "node:fs" "node:fs/promises" "node:os" "node:path" "node:process" "node:readline" "node:stream" "node:string_decoder" "node:timers/promises" "node:tty" "node:url" "node:util" "node:util/types" 
Elapsed: 80902ms | User: 687ms | Sys: 250ms
RSS: 0.20GB | Peak: 0.20GB | Commit: 0.26GB | Faults: 87867

panic(main thread): Segmentation fault at address 0x5
oh no: Bun has crashed. This indicates a bug in Bun, not your code.

**Comment by nektro** [comment]
```
npm create astro@latest
cd .\my-app\
npx astro add solid
npx astro add tailwind
C:\bun\build\debug\bun-debug.exe --bun x solidui-cli@latest init
```

```
┌   solidui-cli - 0.7.0 
│
◇  Would you like to use Typescript (recommended)?
│  Yes
│
◇  Where is your global CSS file? (this file will be overwritten)
│  src/app.css
│
◇  Where is your Tailwind config located? (this file will be overwritten)
│  tailwind.config.cjs
│
◇  Are you using a custom tailwind prefix eg. tw-? (Leave blank if not)
│  undefined
│
◇  Configure the import alias for components:
│  ~/components/ui
│
◇  Configure the import alias for utils:
│  ~/lib/utils
│
◇  ui.config.json created.
│
◇  Project initialized.
============================================================
Bun Debug v1.1.31 (5e8ca1cc) Windows x64
Windows v.win10_fe
CPU: sse42 avx avx2 avx512
Args: "C:\bun\build\debug\bun-debug.exe" "--bun" "x" "solidui-cli@latest" "init"
Features: Bun.stdin(2) Bun.stdout jsc spawn transpiler_cache(13) tsconfig(2) 
Builtins: "bun:jsc" "bun:main" "node:assert" "node:buffer" "node:child_process" "node:crypto" "node:events" "node:fs" "node:fs/promises" "node:module" "node:os" "node:path" "node:process" "node:readline" "node:stream" "node:stream/promises" "node:string_decoder" "node:timers/promises" "node:tty" "node:url" "node:util" "node:util/types" "node:v8" 
Elapsed: 29314ms | User: 22937ms | Sys: 843ms
RSS: 0.51GB | Peak: 0.55GB | Commit: 0.67GB | Faults: 202405

panic(main thread): Segmentation fault at address 0x5
???:?:?: 0x17b8000426d in ??? (???)
???:?:?: 0x17b801d3105 in ??? (???)
???:?:?: 0x100017bc9c691e9 in ??? (???)
???:?:?: 0x17bffdf68c7 in ??? (???)
???:?:?: 0x68e81a55bf in ??? (???)
???:?:?: 0x6 in ??? (???)
???:?:?: 0xa4de53b0b652 in ??? (???)
???:?:?: 0x17b800011b7 in ??? (???)
???:?:?: 0x68e81a564f in ??? (???)
???:?:?: 0x17b801d3072 in ??? (???)
```

**Comment by heimskr** [comment]
Probably fixed by #15357.
